src/js/tech/flash.js

```diff
@@ -26,9 +26,6 @@ class Flash extends Tech {
 
     let { source, parentEl } = options;
 
-    // Create a temporary element to be replaced by swf object
-    let placeHolder = this.el_ = Lib.createEl('div', { id: player.id() + '_temp_flash' });
```

Comment:
I tried to guess why we were using this placeholder and I ended up looking at 2 years old commits. My only guess is that is was needed in the iframe mode. It seems to work perfectly fine without using it. @heff any clue?